armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -124,6 +126,8 @@ def __init__(self, detailedAxialExpansion: bool = False):
         self._detailedAxialExpansion = detailedAxialExpansion
         self.linked = None
         self.expansionData = None
+        self.massConservationReport = None
+        self.detailedMassConservationReport = None
```

Comment:
It doesn't appear that there is any need to store these items on `self`, since they're only really used in one method of this class.

I think that trashing these dicts after each call to `axiallyExpandAssembly()` would be both safe and more memory efficient.

Reply:
I put them on `self` in https://github.com/terrapower/armi/pull/1338/commits/29c345e117746a5a4e69193153697d964f8e92d4 simply to accommodate testing. I could use the mock logs to rebuild them if you'd prefer... But that seemed clunky to me and the easier solution was just to make them class variables.

Reply:
That said, if we kept them as class instances, we could empty them after they are populated. Maybe a compromise of sorts?

Reply:
Ok, so this should be resolved with https://github.com/terrapower/armi/pull/1338/commits/027db327772541229a33290a8116c7789109b807. All of the logging now lives in its own class and only gets populated if the `runLog` verbosity is either `extra` or `debug`. The dicts that get populated will go out of scope with the rest of the expansion changer and should follow suit with the expansion data, etc.